packages/bpk-component-barchart/src/utils-test.js

```diff
@@ -47,16 +47,16 @@ describe('utils', () => {
 
```

Comment:
Are the changes in this file due to the decrease in the number of items in `data.json`?